Add `AccordionDetails` indentation (#1499)
iTwin/stratakit: apps/website/src/content/docs/components/mui/Accordion.md

- The overall size has been decreased.
- Added [responsive design](#responsive-design) that reorients the marker placement based on container width.

> - Added `markerPlacement` prop to override responsive design and force the [marker placement](#marker-placement).

Reviewer: unrelated suggestion, I think `AccordionSummary` deserves a mention 

```suggestion
- Added [`markerPlacement`](#marker-placement) prop to `AccordionSummary` to override responsive design.
```

(feel free to reject)

Author: No, I like it. Accepted.

> - Added details indentation when the summary marker is left aligned.

Reviewer: ```suggestion
- Added details indentation when the `markerPlacement` is set to `"start"`.
```

> - The default `disableGutters` is now `"true"`.

Reviewer: unrelated typo

```suggestion
- The default `disableGutters` is now `true`.
```
